.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,56 @@
+﻿name: Feature Request
+description: Request a Feature or Enhancement to the Project.
+title: "feature: "
+labels: [ "type/enhancement", "type/feature" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the sections below to help us understand and implement the feature. Before submitting a feature request, please check the [Suggesting Enhancements](https://github.com/MeikelLP/quantum-core-x/blob/master/CONTRIBUTING.md#suggesting-enhancements) section in the contributing guidelines.
+
+  - type: textarea
+    id: what-feature
+    attributes:
+      label: What feature would you like to request?
+      description: A clear and concise description of the feature you would like to request.
+      placeholder: Describe the feature you would like to request.
+
+  - type: textarea
+    id: why-feature
+    attributes:
+      label: Why do you need this feature?
+      description: A clear and concise description of why you need this feature.
+      placeholder: Describe why you need this feature, a good and understandable user story would be appreciated.
```

Comment:
Same as above, do we have good examples for these? That goes a long way, and is perfectly in line with keeping requirements low for newcomers while providing good guidance for them, IMO.

Reply:
Maybe get help from ChatGPT? :)

Reply:
Sadly no - again
I guess we grow this as the project grows